dockstore-webservice/src/main/java/io/dockstore/webservice/core/Image.java

```diff
@@ -61,6 +61,10 @@
     @ApiModelProperty(value = "Docker ID of the image", position = 4)
     private String imageID;
 
+    @Column()
+    @ApiModelProperty(value = "Registry image belongs to", position = 5)
```

Comment:
Real nit-picky, but maybe `Registry the image belongs to`? At first glance I read `Registry image` as noun phrase